PowerEditor/installer/nsisInclude/gobalDef.nsh

```diff
@@ -34,6 +34,12 @@
 !define VERSION_MAJOR 7
 !define VERSION_MINOR 0
 
+!define CompanyName		"Don HO don.h@free.fr"
+!define Description		"Notepad++ : a free (GNU) source code editor"
+!define Version 		"7.0.0.0"
```

Comment:
However, I was thinking why can't we remove macro APPVERSION (VERSION_MAJOR and VERSION_MINOR as well). Instead of changing its value for every release, just read the version info from notepad++.exe and use it.

This requires an additional nsis file (get version of npp exe and write version info in normal text file in form of macro with required format.) and its output (exe) file.
Later include call this exe in beginning of nsi script to generate text file and use these defined macro.
We have used this approach in many project. This will reduce the manual effort of writing version info everytime.